modules/utilities/unix/smb/smbclient/secgen_metadata.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0"?>
+
+<utility xmlns="http://www.github/cliffe/SecGen/utility"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://www.github/cliffe/SecGen/utility">
+  <name>Samba client</name>
+  <author>example42</author>
```

Comment:
I must have copied the secgen_metadata.xml from another module and forgot to remove this author. Good spot.